### ISSUE TITLE
fatal: undefined method CRM_Utils_System::loadUser() on CiviCRM 4.6

### ISSUE THREAD

**Issue by xurizaemon** [civicrm/cv#11]
Not sure if `cv` is supported on 4.6, but if I use

    cv api -U admin job.fetch_activities

I get

    PHP Fatal error:  Call to undefined method CRM_Utils_System::loadUser() in phar:///opt/buildkit/bin/cv/src/Command/BaseCommand.php on line 66
    PHP Stack trace:
    PHP   1. {main}() /opt/buildkit/bin/cv:0
    PHP   2. require() /opt/buildkit/bin/cv:10
    PHP   3. Civi\Cv\Application::main() phar:///opt/buildkit/bin/cv/bin/cv:22
    PHP   4. Symfony\Component\Console\Application->run() phar:///opt/buildkit/bin/cv/src/Application.php:17
    PHP   5. Symfony\Component\Console\Application->doRun() phar:///opt/buildkit/bin/cv/vendor/symfony/console/Application.php:124
    PHP   6. Symfony\Component\Console\Application->doRunCommand() phar:///opt/buildkit/bin/cv/vendor/symfony/console/Application.php:193
    PHP   7. Symfony\Component\Console\Command\Command->run() phar:///opt/buildkit/bin/cv/vendor/symfony/console/Application.php:850
    PHP   8. Civi\Cv\Command\ApiCommand->execute() phar:///opt/buildkit/bin/cv/vendor/symfony/console/Command/Command.php:257
    PHP   9. Civi\Cv\Command\BaseCommand->boot() phar:///opt/buildkit/bin/cv/src/Command/ApiCommand.php:48

Same command works fine on 4.7, and maybe relates to the newly added `-U` switch? [Method exists in 4.6 branch](https://github.com/civicrm/civicrm-core/blob/4.6/CRM/Utils/System/Drupal.php#L380) so possibly just a `require` away.

v0.1.5

**Comment by totten** [comment]
I think the difference is that 4.7's version of `CRM_Utils_System` has this:

```php
  public static function __callStatic($name, $arguments) {
    $userSystem = CRM_Core_Config::singleton()->userSystem;
    return call_user_func_array(array($userSystem, $name), $arguments);
  }
```

It's easy enough to work-around (bd4b7b8a0cba6bf63f9f6a2bb82aa29ec137c894), and it's fixed in v0.1.14.

I wasn't sure how to add this to the test suite because `-U` depends on having an example user in the CMS. However, maybe the test could use `cv vars:show` to get `ADMIN_USER` and then use `cv -U $ADMIN_USER php:eval ...` to try it out?

**Comment by eileenmcnaughton** [comment]
4.6 is dead - this can be closed
